xml/System.Net/ServicePointManager.xml

```diff
@@ -756,9 +756,11 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This property selects the version of the Secure Sockets Layer (SSL) or Transport Layer Security (TLS) protocol to use for new connections that use the Secure Hypertext Transfer Protocol (HTTPS) scheme only; existing connections are not changed.  
+This property selects the version of the Secure Sockets Layer (SSL) or Transport Layer Security (TLS) protocol to use for new connections that use the Secure Hypertext Transfer Protocol (HTTPS) scheme only; existing connections are not changed.  
```

Comment:
I don't think this is true: `SecurityProtocol` applies to HttpWebRequest, FtpWebRequest and SmtpClient in .Net Framework. (Certain limitations exist for CoreFX.)

/cc @davidsh, @stephentoub